### PR TITLE
logging improvements + minimuxer swift-bridge refactor

### PR DESCRIPTION
**Pull request by naturecodevoid**
### Changes

- refactor minimuxer to use swift-bridge
- add libi + usbmuxd debug logging and Export Logs button (logs also go to Documents/sidestore.log so they should be accessible even after expire)
- add commit SHA to PR build version 

### Todo before merge

- [x] ensure minimuxer issues are fixed

### CODE REVIEW

**Comment by Spidy123222** [comment]
I don’t see any problems fro marring many users in the discord.

**Comment by JoeMatt** [comment]
My only problem with this is #278 completely outdates this change already, and I'm nearly done with it.

**Comment by JoeMatt** [comment]
closing after looking more into this, my other pr and this will be too hard to merge and i don't think there's anything i missed.

We can re-evaluate after #278 is merged.

File: AltStore/AppDelegate.swift [hunk 1]
```diff
@@ -58,6 +58,34 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool
     {
+        // Copy STDOUT and STDERR to the logging console
```
Comment (JoeMatt) [inline AltStore/AppDelegate.swift:61]: we won't need this since I've added os.log and/or swift-log support already and we can use that to do file writes.

**Pull request closed without merging**